src/js/engagement_view/src/types/GraphDisplayTypes.tsx

```diff
@@ -8,9 +8,11 @@ export type GraphDisplayProps = {
 export type GraphDisplayState = {
     graphData: VizGraph;
     curLensName: string | null;
+    toggle: boolean;
```

Comment:
I guess I'm not entirely sure what this bool represents?

Reply:
A comment would help. It's a boolean that lets us explicitly change the state, which will trigger the useEffect.